book/website/collaboration/leadership/leadership-building.md

```diff
@@ -0,0 +1,103 @@
+They are aware of their own biases and avoid them intentionally, often by providing space to listen to different thoughts and perspectives. 
+They guide people to adapt to changes and transformations particularly when leading large projects, with many moving parts and teams.
+- **Delegation**: Leaders are to lead by example, but they will be unable to do everything and should not do everything either. 
+When a leader is in charge of too many aspects of a project they will become a bottleneck and blocker to the emergence of other open leaders.
+Delegation goes hand in hand with building a trustworthy environment, this means that they communicate clearly the overarching goals, create smaller milestones and delegate tasks meaningfully so different people can lead different parts of the work, nurturing healthy leadership habits within their teams.
+- **Power dynamics**: Power dynamics are relationships between people in an organisation with varying levels of privileges. 
+Power dynamics are often created by leaders.
+Leaders with formal authority come with institutional power and open leaders often come with informal connections that can translate into social capital or power (even when they don’t have formal authority in the organisation).
+Therefore, leaders should understand how their positions affect the dynamics when working with others in their team, even when not exerting their power.
+Understanding power dynamics helps leaders identify and resolve conflict, build consensus, and create a more effective team culture.
+- **Acknowledging fairly**: Leadership roles come with a lot of visibility for their roles.
+This is partly earned by leaders' work as individuals but is often enhanced by the work of others in their teams. 
+Open leaders recognise and celebrate all kinds of work, appreciate and celebrate them openly and give appropriate credit to everyone, including those whose work makes them visible often taking invisible responsibilities for the sake of the overall success.
+
+##  What should Leaders Avoid
+
+- **Elitism and hypocrisy**: Open leadership in research in general benefits a lot from equity, inclusivity and diversity (EDI). 
+Thus, traits that sabotage EDI goals, should be avoided at all costs.
+Attitudes like elitism and hypocrisy can have unintended consequences on the project as well as on the leaders themselves.
+Elitism is when leaders constantly reinforce their authority, status, and expertise, creating distance between them and junior/new members of their teams.
+Hypocrisy is holding others accountable for one set of quality measures, and having a lesser standard for themselves.
+Leaders should hold the same standards for everyone and create spaces for collaboration through egalitarian rather than authoritative means.
+- **Ignoring the personal side of the team**: members of the team/organisation can disclose or not as much as they want from their personal lives. 
```

Comment:
I think we should start by acknowledging that people have multiple sides of their lives and only part of it might be visible when they contribute to a project or team, then go into the rest of this

Reply:
```suggestion
- **Ignoring the personal side of the team**: People have multiple sides of their lives and only part of it might be visible when they contribute to a project or team.
Leaders should set an environment where their team members feel safe to disclose as much or as little they want from their personal lives. 
```